### DISCUSSION TITLE
support for NXP i.MX 8M PLUS

### DISCUSSION THREAD

**Discussion by krishnarao21** [adafruit/Adafruit_Blinka#1085] [New Board Requests]
Hi 
I am using a board which is based on NXP i.MX8M PLUS. 
Getting the following details with Detector.
Chip id: IMX8MP
Board id: None

With this unable to import board (i.e. getting NotImplementedError).

Installed required packages Adfruit-Blinka, Adfruit-circuitpython-bno08x, Adfruit-circuitpython-busdevice ...

How to use the i2c interface of the processor gpio with current packages to integrate with the bno08x? is it possible?

**Comment by makermelissa** [comment]
Detection for the board will need to be added to https://github.com/adafruit/Adafruit_Python_PlatformDetect. If you'd like to add it yourself, you can follow this guide: https://learn.adafruit.com/adding-a-single-board-computer-to-platformdetect-for-blinka. Otherwise, please provide the board name.

**Comment by krishnarao21** [comment]
Hi 
changed /usr/lib/python3.11/site-packages/adafruit_platformdetect/board.py to detect the board (i.e. NXP_IMX8MPLUS_SOM),
now getting the error from Adafruit_Blinka package 
File "/usr/lib/python3.11/site-packages/board.py", line 439, in <module>
    raise NotImplementedError(f"Board not supported {board_id}.")
NotImplementedError: Board not supported NXP_IMX8MPLUS_SOM. 

There is no folder/file related to NXP in https://github.com/adafruit/Adafruit_Blinka/tree/main/src/adafruit_blinka/board
Please provide support for NXP IMX8MPLUS board, if already provided, please provide me the istructions for use.

**Comment by krishnarao21** [comment]
https://www.karo-electronics.com/fileadmin/download/Datasheets/QSXP.pdf
Our board is Karo based SOM
